xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -32,6 +32,12 @@ class CPVRClient;
 class CPVRProvider;
 class CPVRTimerInfoTag;
 
+enum class RecordingMediaType : bool
+{
+  PVR_RECORDING_MEDIA_TYPE_RECORDING,
+  PVR_RECORDING_MEDIA_TYPE_VOD
+};
```

Comment:
PVR_RECORDING_MEDIA_TYPE_UNKNOWN and implementation missing.

Reply:
Not relevenat, media type will always be known.